### PR TITLE
Refactor to mvvm

### PR DESCRIPTION
**Pull request by erikdvt**
Refactored codebase to MVVM.
Changed query parameters to return top 25 latest marvel comics.

### CODE REVIEW

File: Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift [hunk 1]
```diff
@@ -0,0 +1,27 @@
+//
+//  ComicTableViewCellExtension.swift
+//  Marv-Lous
+//
+//  Created by Erik Egers on 2022/03/31.
+//
+
+import Foundation
+
+extension ComicTableViewCell {
```
Comment (bgouws) [inline Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift]: So maybe an enhancement we can make here is how about we extend `String`?  then when you needed to access these functions you could do something like this: 

`link.convertToHttps` 😄 let me know if you have questions.

File: Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    func convertToHttps(httpURL: String) -> String {
+        var httpsURL = httpURL
+        let index = httpsURL.index(httpsURL.startIndex, offsetBy: 4)
+        
+        httpsURL.insert("s", at: index)
+        return httpsURL
+    }
+    
+    func buildImageURL(thumbnailPath: String, thumbnailExtension: String) -> String {
```
Comment (bgouws) [inline Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift]: Same with this one 😄

File: Marv-Lous/Marv-Lous/Repositories/Models/ComicsModel.swift [hunk 3]
```diff
@@ -8,7 +8,7 @@
 import Foundation 
 
 // MARK: - ComicDataWrapper
-struct ComicDataWrapper: Codable {
+struct ComicsModel: Codable {
```
Comment (bgouws) [inline Marv-Lous/Marv-Lous/Repositories/Models/ComicsModel.swift:11]: Nice! 😄

**Review by bgouws — CHANGES_REQUESTED** [review]
Really excellent PR @erikdvt 👏 just a couple of small enhancements but this looks really great 👍

File: Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift (hunk 1 above)
Comment (erikdvt) [reply Marv-Lous/Marv-Lous/Extensions/ComicTableViewCellExtension.swift]: I managed to figure this out.

**Review by bgouws — APPROVED** [review]
Very nice!

**Review by kieranwoodrow — APPROVED** [review]

**Pull request merged**